engine/src/test/resources/application.conf

```diff
@@ -1,44 +1,139 @@
+  abort-jobs-on-terminate = false
 }
 
 workflow-options {
+  // These workflow options will be encrypted when stored in the database
   encrypted-fields: ["refresh_token"]
+
+  // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
   workflow-log-dir: "cromwell-test-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
   workflow-log-temporary: true
+
+  // Workflow-failure-mode determines what happens to other calls when a call fails. Can be either ContinueWhilePossible or NoNewCalls.
+  // Can also be overridden in workflow options. Defaults to NoNewCalls. Uncomment to change:
+  //workflow-failure-mode: "ContinueWhilePossible"
+}
+
+// Optional call-caching configuration. Disabled by default.
+call-caching {
+  enabled = false
+
+  // The Docker image specified in the 'runtime' section of a task can be used as-is
+  // or Cromwell can lookup this Docker image to get a complete hash.  For example,
+  // if a task specifies docker: "ubuntu:latest" and if lookup-docker-hash is true,
+  // Then Cromwell would query DockerHub to resolve "ubuntu:latest" to something like
+  // a2c950138e95bf603d919d0f74bec16a81d5cc1e3c3d574e8d5ed59795824f47
+  //
+  // A value of 'true' means that call hashes will more accurately represent the
+  // Docker image that was used to run the call, but at a cost of having to make a
+  // request to an external service (DockerHub, GCR).  If a call fails to lookup a
+  // Docker hash, it will fail.
+  lookup-docker-hash = false
 }
 
 google {
+
   application-name = "cromwell"
 
   auths = [
     {
-      name = "cromwell-system-account"
+      name = "application-default"
       scheme = "application_default"
-    }
-    {
-      name = "cromwell-service-account"
-      scheme = "service_account"
-      service-account-id = "my-cromwell-account"
-      pem-file = "/path/to/pem/file"
-    }
+    },
     {
       name = "user-via-refresh"
       scheme = "refresh_token"
-      client-id = "secret-id"
-      client-secret = "secret-secret"
+      client-id = "secret_id"
+      client-secret = "secret_secret"
+    },
+    {
+      name = "service-account"
+      scheme = "service_account"
+      service-account-id = "my-service-account"
+      pem-file = "/path/to/file.pem"
     }
   ]
 }
 
 backend {
   default = "Local"
-  providers = [
-    {
-      name = "Local"
+  providers {
+    Local {
       class = "cromwell.engine.backend.local.LocalBackend"
+      actor-factory = "cromwell.backend.impl.local.LocalBackendLifecycleActorFactory"
+      config {
+        // Root directory where Cromwell writes job results.  This directory must be
+        // visible and writeable by the Cromwell process as well as the jobs that Cromwell
+        // launches.
+        root: "cromwell-executions"
+
+        filesystems = {
```

Comment:
There's no difference between `=`, `:`, and space. HOCON is to JSON as HTML is to XML. Just like when one writes html you can use `<br>` instead of `<br/>`, HOCON gives config files a lot of flexibility on style. IMHO, just keep it consistent. The reason `=` and `:` are both allowed is because HOCON attempts to be a superset of both JSON and ([mostly](https://github.com/typesafehub/config/blob/master/HOCON.md#note-on-java-properties-similarity)) Java's Property Files.



> Key-value separator
> The = character can be used anywhere JSON allows :, i.e. to separate keys from values.
> If a key is followed by {, the : or = may be omitted. So "foo" {} means "foo" : {}

https://github.com/typesafehub/config/blob/master/HOCON.md#key-value-separator

The reason you're seeing different optional delimiters is because different coders/reviewers let different things pass. In the end, like my other prefs in this PR, it really doesn't matter to the correctness of your patch.